cockpit/de.json

```
@@ -358,8 +363,18 @@
     "VARIABLE_SERIALIZED_VALUE": "Serialisierter Wert",
     "TIME_TO_LIVE_DAY": "{{ttl}} Tag",
     "TIME_TO_LIVE_DAYS": "{{ttl}} Tage",
-    "TIME_TO_LIVE_MESSAGE_ERR": "Fehler beim Aktualisieren des Lebenszyklus historischer Daten",
+    "TIME_TO_LIVE_MESSAGE_ERR": "Fehler beim Aktualisieren des Lebenszyklus historischer Daten.",
     "TIME_TO_LIVE_NULL": "null",
+    "TIME_TO_LIVE_POPUP_TITLE": "Wert des Lebenszyklus historischer Daten ändern",
+    "TIME_TO_LIVE_POPUP_SUBTITLE": "Option wählen:",
+    "TIME_TO_LIVE_POPUP_RADIO_UPDATE": "Wert aktualisieren",
+    "TIME_TO_LIVE_POPUP_RADIO_UPDATE_TOOLTIP": "Aktualisiert den Wert mit dem jeweilig gewählen",
```

I think a _t_ is missing.

```suggestion
    "TIME_TO_LIVE_POPUP_RADIO_UPDATE_TOOLTIP": "Aktualisiert den Wert mit dem jeweilig gewählten",
```

---

👍